gradio/blocks.py

```diff
@@ -2252,9 +2276,15 @@ def get_api_info(self):
         """
         config = self.config
         api_info = {"named_endpoints": {}, "unnamed_endpoints": {}}
-        mode = config.get("mode", None)
 
-        for d, dependency in enumerate(config["dependencies"]):
+        for dependency in config["dependencies"]:
+            if (
+                not dependency["backend_fn"]
```

Comment:
I think this change does not account for tabbed interfaces any longer and so the tabbed interface api docs are broken

<img width="1440" alt="image" src="https://github.com/gradio-app/gradio/assets/41651716/6a8e6e65-4511-4c5d-9f7d-cea6b429e58b">


I ran the `stt_or_tts` demo

Reply:
Interesting, nice catch will take a look

Reply:
I can't repro this, perhaps it was fixed in my last change -- would you be able to double check if its still an issue for you @freddyaboulton?